.env.example

```
@@ -1,5 +1,5 @@
-#secure password, can use openssl rand -hex 32
-NUXT_SESSION_PASSWORD=""
+# secure password, can use openssl rand -hex 32
+NUXT_SESSION_PASSWORD=
 
-#HMAC secret for image-proxy and OG image URL signing, can use openssl rand -hex 32
-NUXT_IMAGE_PROXY_SECRET=""
\ No newline at end of file
+# HMAC secret for image-proxy and OG image URL signing, can use openssl rand -hex 32
+NUXT_IMAGE_PROXY_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)